src/BBT.StructureTools/Compare/IComparer.cs

```diff
@@ -0,0 +1,40 @@
+﻿// Copyright © BBT Software AG. All rights reserved.
+
+namespace BBT.StructureTools.Compare
+{
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Interface for model comparison.
+    /// ---------------------------------
+    /// Remark for hash code calculation:
+    /// ---------------------------------
```

Comment:
Use XML comment tags for formatting. We'll generate a website documenting the API from these comments (beside using them for IntelliSense)